b2b-sample/README.md

```diff
@@ -131,16 +124,23 @@ To create a user for Best Auto Mart with permissions to create an IdP for the **
         "name": "Guardio Insurance - Administrator Application",
         "tag": "Administrator Application"
       }
+    },
+    "ManagementAPIConfig": {
+      "SharedApplicationName": "Guardio-Business-App",
+      "UserStore" : "DEFAULT"
     }
   }
 }
 ```
 setup the config json file as mentioned above.
 
-### Step 5: Additional settings
+**_NOTE:_**  With the latest B2B organization management features provided by WSO2 Identity Server, the following operations are allowed in sub organization for the organization creator.
+
+1 - onboarding enterprise IDP.
+2 - Update authentication options of the shared applications.
+3 - Delete the business users.
```

Comment:
With the latest features, above operations can be done from the Console app itself which exceeds the capabilities of the b2b sample app. In order to not to limit the functionalities which can be done at the sub organization by the less features of the b2b sample app, we can suggest to check the Console app.
ex - configure multiple authentication steps / options
     - delete users